### ISSUE TITLE
did \m ever get merged? 

### ISSUE THREAD

**Issue by dustymabe** [calid/bash#4]
In your description you say you don't know if it will ever get merged. Did it? Is there a reason for not merging it?

**Comment by calid** [comment]
No, doesn't look like it.  I'm not sure Chet's reason for not merging it.  I followed up a couple times but he never really answered me.  You can read the last correspondence here: http://lists.gnu.org/archive/html/bug-bash/2015-12/msg00031.html

I'd encourage you and anyone else who wants to see this functionality in the mainline to email the bash list and request it.  Strength in numbers and all that.

Good luck!

**Comment by calid** [comment]
on second thought I'll leave this issue open, and if `\m` ever gets merged we can celebrate by closing it :)
